tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCheckCLDR.java

```diff
@@ -73,6 +73,9 @@
 import com.ibm.icu.text.UnicodeSet;
 
 public class TestCheckCLDR extends TestFmwk {
+
+    private static final boolean SHOW_LIMITED = System.getProperty("TestCheckCLDR:SHOW_LIMITED") != null;
+
     static CLDRConfig testInfo = CLDRConfig.getInstance();
     private final Set<String> eightPointLocales = new TreeSet<>(
         Arrays.asList("ar ca cs da de el es fi fr he hi hr hu id it ja ko lt lv nl no pl pt pt_PT ro ru sk sl sr sv th tr uk vi zh zh_Hant".split(" ")));
```

Comment:
I though Bangla (bn) was also a grammar locale?

Reply:
Ok, are we intentionally excluding indic locales from Meike's sheet? If so we should update that sheet.